packages/bot-web-ui/src/components/dashboard/bot-notification.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Toast } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import RootStore from 'Stores/root-store';
+import { connect } from 'Stores/connect';
+
+type TBotNotification = {
+    show_toast: boolean;
+    toast_message: string;
+    setOpenSettings: (toast_message: string, show_toast: boolean) => void;
+};
+
+const BotNotification = ({ show_toast, toast_message, setOpenSettings }: TBotNotification) => {
+    React.useEffect(() => {
+        setTimeout(() => {
+            setOpenSettings(toast_message, false);
+        }, 3000);
+    }, [show_toast]);
+    const el_portal = document.getElementById('popup_root');
+
+    if (el_portal) {
+        return ReactDOM.createPortal(
+            <div className='import-notification'>
+                {show_toast && (
+                    <div>
+                        <Toast>
+                            {toast_message === 'delete'
+                                ? localize('You’ve successfully deleted a bot.')
+                                : localize('You’ve successfully imported a bot.')}
+                        </Toast>
+                    </div>
+                )}
+            </div>,
+            el_portal
+        );
+    }
```

Comment:
Can we not render and create portal when `show_toast` is false? Maybe we can add the condition in the upper scope.

Reply:
Sure. Done ✅